lib/galaxy/tool_util/loader_directory.py

```diff
@@ -171,7 +171,10 @@ def looks_like_xml(path, regex=TOOL_REGEX):
         return False
 
     with open(path, "r") as f:
```

Comment:
```suggestion
    with io.open(path, "r", encoding='utf-8') as f:
```

with adding `import io` at the top.

Reply:
I don't think we want to do this here, it prevents people from choosing a different locale.

Reply:
With are already using this in many other places, including `lib/galaxy/tool_util/verify/__init__.py` . I think it solves more issues (e.g. strange users' locales) than it creates.